<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GanttWidget</class>
 <widget class="QWidget" name="GanttWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GanttWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="GanttIntervalSliderZoom" name="intervalSlizedZoom" native="true"/>
     </item>
     <item>
      <widget class="GanttIntervalSlider" name="intervalSlider" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="GanttPlayerSettings" name="playerSettings" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="GanttPlayerControl" name="playerControl" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="GanttTreeView" name="treeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="GanttGraphicsView" name="ganttView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GanttPlayerSettings</class>
   <extends>QWidget</extends>
   <header>ganttplayersettings.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GanttIntervalSliderZoom</class>
   <extends>QWidget</extends>
   <header>ganttintervalsliderzoom.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GanttTreeView</class>
   <extends>QTreeView</extends>
   <header>gantttreeview.h</header>
  </customwidget>
  <customwidget>
   <class>GanttGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>ganttgraphicsview.h</header>
  </customwidget>
  <customwidget>
   <class>GanttIntervalSlider</class>
   <extends>QWidget</extends>
   <header>ganttintervalslider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GanttPlayerControl</class>
   <extends>QWidget</extends>
   <header>ganttplayercontrol.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
